app/views/security_events/_table.html.erb

```diff
@@ -0,0 +1,84 @@
+<%# locals:
+  - security_events
+  - next_page (optional)
+  - prev_page (optional)
+  - show_user (optional)
+%>
+
+<div class="grid-container padding-0">
+  <div class="grid-row">
+    <table class="usa-table grid-col-fill">
+      <thead>
+        <tr>
+          <% if local_assigns[:show_user] %>
+            <th scope="col">User</th>
+          <% end %>
+          <th scope="col">Type</th>
+          <th scope="col">UUID</th>
+          <th scope="col">Issued At</th>
+          <th scope="col">Payload</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% security_events.each do |security_event| %>
+          <tr>
+            <% if local_assigns[:show_user] %>
+              <td>
+                <%= security_event.user.email %>
+              </td>
+            <% end %>
+            <%= content_tag(:td, title: security_event.event_type) do %>
+              <%= security_event.event_type.split('/').last %>&hellip;
+            <% end %>
+            <td>
+              <kbd><%= security_event.uuid %></kbd>
+            </td>
+            <td>
+              <%= l(security_event.issued_at, format: :long) %>
+            </td>
+            <td>
+              <% if security_event.raw_event %>
+                <details>
+                  <summary class="summary-fix">Show Payload</summary>
+                  <pre><code><%= JSON.pretty_generate(JSON.parse(security_event.raw_event)) %></code></pre>
+                </details>
+              <% end %>
```

Comment:
Question maybe for @aduth, right now when the summary expands, it blows up the size of the entire table because it changes the column widths. Do you have any other ideas for showing the full details? I guess I could do pairs of rows and have a `colspan` kinda thing going so that it expands downwards only?

Reply:
> Question maybe for @aduth, right now when the summary expands, it blows up the size of the entire table because it changes the column widths. Do you have any other ideas for showing the full details? I guess I could do pairs of rows and have a `colspan` kinda thing going so that it expands downwards only?

That could work, yeah. Maybe paired with an expanding control, like in [this example](https://css-tricks.com/table-with-expando-rows/).

I think a simpler alternative could be to create a dedicated detail page for each event, and include the payload on that.

Reply:
Good point, will go with the detail page approach

Reply:
Added details page in 5a31759d9575acc5f3ea99eb2d23254de16fc1ee

<img width="778" alt="Screen Shot 2021-04-13 at 9 08 24 AM" src="https://user-images.githubusercontent.com/458784/114584903-f1df2700-9c37-11eb-8a11-96c52681cef7.png">